output.tf

```diff
@@ -0,0 +1,19 @@
+output "efs_file_system_arn" {
+  value = aws_efs_file_system.efs_file_system.*.arn
```

Comment:
remove splat expression from output, splat use if you are getting a list of elements.